apps/guardian-ui/src/languages/en.json

```diff
@@ -0,0 +1,123 @@
+{
+  "common": {
+    "next": "Next",
+    "back": "Back"
+  },
+  "app": {
+    "error": "Something went wrong."
```

Comment:
This could probably in `common`. We use it several times. Hopefully our error handling improves from this point. 😅